Containerfile

```diff
@@ -7,13 +7,18 @@ RUN ansible-galaxy collection install -r /build/requirements.yml --collections-p
 
 RUN mkdir -p /usr/share/ansible/roles /usr/share/ansible/collections
 
+FROM quay.io/ansible/python-builder:latest as builder
+
+ADD requirements_combined.txt /tmp/src/requirements.txt
+ADD bindep_combined.txt /tmp/src/bindep.txt
+RUN assemble
+
 FROM quay.io/ansible/ansible-runner:devel
+
 RUN pip3 install --upgrade pip setuptools
```

Comment:
this is redundant now, but we can fix later.

Reply:
+1, this will require a change in builder itself.

Reply:
oops. nvm. it's in the definition!

Reply:
from tools/execution-environment.yaml prepend.

Reply:
addressed in latest commit.